frontend/elements/src/template-popups.ts

```diff
@@ -0,0 +1,51 @@
+import { MEDIA_UI } from '@utils/path';
```

Comment:
if `template-popups`  is a reusable thing, it should be in `core`.

However, I don't know what `template` means... can we find a better name for what this is?

Reply:
this has been moved to the right folder